lib/Myriad.pm

```diff
@@ -587,6 +589,20 @@ method setup_tracing () {
     return;
 }
 
+=head2 tell_parent
+
+Send a message to the parent process
+in case if Myriad is running under
+debug mode.
+
+=cut
+
+method tell_parent ($data) {
+    if ($parent_pipe) {
+        print $parent_pipe "$data\r\n";
```

Comment:
`$parent_pipe` is non-blocking. What if you get only a partial write? Maybe using `syswrite` and handling the conditions would be appropriate?